src/log/zipkin.cpp

```diff
@@ -14,13 +14,15 @@
 
 namespace fc {
 
+std::atomic<bool> sighup_requested = false;
```

Comment:
I think this would be better in an anonymous namespace.

Reply:
Fixed.